core/ui/compose/designsystem/src/test/kotlin/app/k9mail/core/ui/compose/designsystem/atom/textfield/TextFieldOutlinedPasswordTest.kt

```diff
@@ -1,20 +1,26 @@
 package app.k9mail.core.ui.compose.designsystem.atom.textfield
 
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.testTag
 import androidx.compose.ui.test.SemanticsNodeInteraction
 import androidx.compose.ui.test.SemanticsNodeInteractionsProvider
 import androidx.compose.ui.test.assertIsDisplayed
 import androidx.compose.ui.test.onNodeWithContentDescription
+import androidx.compose.ui.test.onNodeWithTag
 import androidx.compose.ui.test.onNodeWithText
 import androidx.compose.ui.test.performClick
+import androidx.compose.ui.test.performTextInput
 import app.k9mail.core.ui.compose.designsystem.R
 import app.k9mail.core.ui.compose.testing.ComposeTest
 import assertk.assertThat
+import assertk.assertions.isEqualTo
 import assertk.assertions.isTrue
 import org.junit.Test
 
 private const val PASSWORD = "Password input"
+private const val TEST_TAG = "TextFieldOutlinedPassword"
 
-class TextFieldOutlinedPasswordKtTest : ComposeTest() {
```

Comment:
The `Kt` in the name is necessary to easily jump from a composable function to it's test using `Command` + `Shift` + `T` (mac). Without it the IDE doesn't find the test.